src/Countries/US/Oregon/WorkersCompAssessmentFund/V20190101/WorkersCompAssessmentFund.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Oregon\WorkersCompAssessmentFund\V20190101;
+
+use Appleton\Taxes\Countries\US\Oregon\WorkersCompAssessmentFund\WorkersCompAssessmentFund as BaseWorkersCompAssessmentFund;
+use Illuminate\Database\Eloquent\Collection;
+
+class WorkersCompAssessmentFund extends BaseWorkersCompAssessmentFund
+{
+    public function compute(Collection $tax_areas)
+    {
+        return 0.0;
```

Comment:
Seems useful.

Reply:
Tell Tasie to stop testing with 2019 data. I added in case there was any left over shifts from 2019 that haven't been paid or are updated and paid out during 2019. May be overkill but I thought it was ok. If I need to remove it I can.